src/extension.ts

```diff
@@ -35,6 +35,15 @@ let configSettings: IAggregateConfiguration;
 let extensionFeatures: IFeature[] = [];
 
 export function activate(context: vscode.ExtensionContext) {
+  let pkg = vscode.extensions.getExtension('jpogran.puppet-vscode');
+  if (pkg){
+    let message = 'The "jpogran.puppet-vscode" extension has been detected. Please uninstall it by executing the following from the commandline: "code --uninstall-extension jpogran.puppet-vscode"';
```

Comment:
Is it worth mentioning why this is bad?

Reply:
```suggestion
    let message = 'The "jpogran.puppet-vscode" extension has been detected. Is not maintained anymore. To uninstall it, execute the following from the commandline: "code --uninstall-extension jpogran.puppet-vscode"';
```
?

Reply:
I think Glenn meant the kind of detail I put in the commit message could be useful to tell the user. It's less important that its unmaintained compared to the race condition that will cause errors.

Reply:
the fact that I missed it in the commit message seems to indicate that glenn is right :-D

Reply:
Updated the text